Index: b/helloDSPgpp/Makefile
===================================================================
--- a/helloDSPgpp/Makefile
+++ b/helloDSPgpp/Makefile
@@ -5,7 +5,7 @@
 #
 #  @desc   Makefile for helloDSPgpp (Debug and Release versions)
 #
-#  @ver    1.10
+#  @ver    1.10p1
 #   ----------------------------------------------------------------------------
 #  Copyright (c) Texas Instruments Incorporated 2002-2009
 #
@@ -18,22 +18,25 @@ SHELL = /bin/sh
 #   Base for toolchain
 #   This may change depending if you are using Montavista or Codesourcery
 #   ----------------------------------------------------------------------------
-BASE_TOOLCHAIN := /opt/mv_pro_5.0/montavista/pro/devkit/arm/v5t_le
+#BASE_TOOLCHAIN := /opt/mv_pro_5.0/montavista/pro/devkit/arm/v5t_le
 #BASE_TOOLCHAIN := /opt/arm/arm-2007q3       # Codesourcery tools
+BASE_TOOLCHAIN := ${CSTOOLS_DIR}
 
 #   ----------------------------------------------------------------------------
 #   Name of the Linux compiler
 #   This may change depending if you are using Montavista or Codesourcery
 #   ----------------------------------------------------------------------------
-CC := arm_v5t_le-gcc
+#CC := arm_v5t_le-gcc
 #CC := arm-none-linux-gnueabi-gcc
+CC := ${TARGET_PREFIX}gcc
 
 #   ----------------------------------------------------------------------------
 #   Base for DSPLink
 #   This may have to be changed depending on
 #   your installation and DSPLink version
 #   ----------------------------------------------------------------------------
-DSPLINK := /home/user/OMAPL137_arm_1_00_00_10/dsplink-1_61_03-prebuilt/packages/dsplink
+#DSPLINK := /home/user/OMAPL137_arm_1_00_00_10/dsplink-1_61_03-prebuilt/packages/dsplink
+DSPLINK := ${LINK_INSTALL_DIR}/dsplink
 
 #   ----------------------------------------------------------------------------
 #   General options, sources and libraries
@@ -51,10 +54,10 @@ BIN := helloDSPgpp
 #   ----------------------------------------------------------------------------
 OBJDIR_D := Debug
 BINDIR_D := $(OBJDIR_D)
-LIBS_D := $(DSPLINK)/gpp/export/BIN/Linux/OMAPL1XX/DEBUG/dsplink.lib $(LIBS)
+#LIBS_D := $(DSPLINK)/gpp/export/BIN/Linux/OMAPL1XX/DEBUG/dsplink.lib $(LIBS)
 # If the DSP/Link was rebuilt by the user, replace the line above 
 # with the one below to use the updated libraries
-#LIBS_D := $(DSPLINK)/gpp/BUILD/EXPORT/DEBUG/dsplink.lib $(LIBS)
+LIBS_D := $(DSPLINK)/gpp/BUILD/EXPORT/DEBUG/dsplink.lib $(LIBS)
 OBJS_D := $(SRCS:%.c=$(OBJDIR_D)/%.o)
 ALL_DEBUG := -g -DDDSP_DEBUG $(DEBUG) -D__DEBUG
 
@@ -63,10 +66,10 @@ ALL_DEBUG := -g -DDDSP_DEBUG $(DEBUG) -D
 #   ----------------------------------------------------------------------------
 OBJDIR_R := Release
 BINDIR_R := $(OBJDIR_R)
-LIBS_R := $(DSPLINK)/gpp/export/BIN/Linux/OMAPL1XX/RELEASE/dsplink.lib $(LIBS)
+#LIBS_R := $(DSPLINK)/gpp/export/BIN/Linux/OMAPL1XX/RELEASE/dsplink.lib $(LIBS)
 # If the DSP/Link was rebuilt by the user, replace the line above 
 # with the one below to use the updated libraries
-#LIBS_R := $(DSPLINK)/gpp/BUILD/EXPORT/RELEASE/dsplink.lib $(LIBS)
+LIBS_R := $(DSPLINK)/gpp/BUILD/EXPORT/RELEASE/dsplink.lib $(LIBS)
 OBJS_R := $(SRCS:%.c=$(OBJDIR_R)/%.o)
 
 #   ----------------------------------------------------------------------------
@@ -75,7 +78,6 @@ OBJS_R := $(SRCS:%.c=$(OBJDIR_R)/%.o)
 INCLUDES := -I$(DSPLINK)/gpp/inc                   \
             -I$(DSPLINK)/gpp/inc/usr               \
             -I$(DSPLINK)/gpp/inc/sys/Linux         \
-            -I$(DSPLINK)/gpp/inc/sys/Linux/2.6.18  \
             -I$(BASE_TOOLCHAIN)/target/usr/include \
             -I./
 
Index: b/helloDSPdsp/Makefile
===================================================================
--- a/helloDSPdsp/Makefile
+++ b/helloDSPdsp/Makefile
@@ -5,7 +5,7 @@
 #
 #  @desc   Makefile for helloDSP (Debug and Release versions)
 #
-#  @ver    1.10
+#  @ver    1.10p1
 #   ----------------------------------------------------------------------------
 #  Copyright (c) Texas Instruments Incorporated 2002-2009
 #
@@ -18,24 +18,29 @@ SHELL = /bin/sh
 #   Base for TI toolchain
 #   This may change depending if you are using other versions of the compiler
 #   ----------------------------------------------------------------------------
-BASE_TOOLCHAIN := /home/user/OMAPL137_arm_1_00_00_10
-BASE_COMPILER := $(BASE_TOOLCHAIN)/cg6x_6_1_9
-BASE_SABIOS := $(BASE_TOOLCHAIN)/bios_5_33_05
-BASE_BSL := $(BASE_TOOLCHAIN)/boards/evmomapl137_v1
+#BASE_TOOLCHAIN := /home/user/OMAPL137_arm_1_00_00_10
+BASE_TOOLCHAIN := ${CSTOOLS_DIR}
+#BASE_COMPILER := $(BASE_TOOLCHAIN)/cg6x_6_1_9
+BASE_COMPILER := ${CODEGEN_INSTALL_DIR}
+#BASE_SABIOS := $(BASE_TOOLCHAIN)/bios_5_33_05
+BASE_SABIOS := ${BIOS_INSTALL_DIR}
+#BASE_BSL := $(BASE_TOOLCHAIN)/boards/evmomapl137_v1
+BASE_BSL := ${EVMOMAPL_BSL_INSTALL_DIR}/evmomapl137_v1
 
 #   ----------------------------------------------------------------------------
 #   Name of the TI compiler and DSP/BIOS builder (Tconf)
 #   This usually does not change
 #   ----------------------------------------------------------------------------
 CC := cl6x
-TCONF := tconf
+TCONF := ${XDC_INSTALL_DIR}/tconf
 
 #   ----------------------------------------------------------------------------
 #   Base for DSPLink
 #   This may have to be changed depending on your installation paths 
 #   and DSPLink version
 #   ----------------------------------------------------------------------------
-DSPLINK := /home/user/OMAPL137_arm_1_00_00_10/dsplink-1_61_03-prebuilt/packages/dsplink
+#DSPLINK := /home/user/OMAPL137_arm_1_00_00_10/dsplink-1_61_03-prebuilt/packages/dsplink
+DSPLINK := ${LINK_INSTALL_DIR}/dsplink
 
 #   ----------------------------------------------------------------------------
 #   Project name. 
@@ -54,10 +59,11 @@ DEBUG :=
 CFLAGS := -I$(BASE_BSL)/dsp/include
 LDFLAGS := -i$(BASE_BSL)/dsp/lib
 CSRCS := tskMessage.c helloDSP_config.c main.c led_test.c
+VPATH += $(BASE_BSL)/dsp/tests/led
 ASRCS :=
 BIOSTCF := helloDSP.tcf
 OBJS :=
-LIBS := -ldsplink.lib -ldsplinkpool.lib -ldsplinkmpcs.lib -ldsplinkmplist.lib -ldsplinkmsg.lib -ldsplinknotify.lib -ldsplinkringio.lib -levmomapl137bsl.lib
+LIBS := -ldsplink.lib -ldsplinkpool.lib -ldsplinkmpcs.lib -ldsplinkmplist.lib -ldsplinkmsg.lib -ldsplinknotify.lib -ldsplinkringio.lib -levmomapl137bsl.lib -ldsplinkdata.lib
 # If the DSP/Link was rebuilt by the user, add -ldsplinkdata.lib to the line above
 CMDS := $(PROJNAME).cmd
 MAPFILE := $(PROJNAME).map
@@ -161,18 +167,18 @@ LDINCLUDES := -i$(BASE_COMPILER)/lib    
 #   ----------------------------------------------------------------------------
 #   Linker library search path for Debug
 #   ----------------------------------------------------------------------------
-LDINCLUDES_D := $(LDINCLUDES) -i$(DSPLINK)/dsp/export/BIN/DspBios/OMAPL1XX/OMAPL1XXGEM_0/DEBUG/
+#LDINCLUDES_D := $(LDINCLUDES) -i$(DSPLINK)/dsp/export/BIN/DspBios/OMAPL1XX/OMAPL1XXGEM_0/DEBUG/
 # If the DSP/Link was rebuilt by the user, replace the line above 
 # with the one below to use the updated libraries
-#LDINCLUDES_D := $(LDINCLUDES) -i$(DSPLINK)/dsp/BUILD/OMAPL1XXGEM_0/EXPORT/DEBUG/
+LDINCLUDES_D := $(LDINCLUDES) -i$(DSPLINK)/dsp/BUILD/OMAPL1XXGEM_0/EXPORT/DEBUG/
 
 #   ----------------------------------------------------------------------------
 #   Linker library search path for Release
 #   ----------------------------------------------------------------------------
-LDINCLUDES_R := $(LDINCLUDES) -i$(DSPLINK)/dsp/export/BIN/DspBios/OMAPL1XX/OMAPL1XXGEM_0/RELEASE/
+#LDINCLUDES_R := $(LDINCLUDES) -i$(DSPLINK)/dsp/export/BIN/DspBios/OMAPL1XX/OMAPL1XXGEM_0/RELEASE/
 # If the DSP/Link was rebuilt by the user, replace the line above 
 # with the one below to use the updated libraries
-#LDINCLUDES_R := $(LDINCLUDES) -i$(DSPLINK)/dsp/BUILD/OMAPL1XXGEM_0/EXPORT/RELEASE/
+LDINCLUDES_R := $(LDINCLUDES) -i$(DSPLINK)/dsp/BUILD/OMAPL1XXGEM_0/EXPORT/RELEASE/
 
 #   ----------------------------------------------------------------------------
 #   Compiler and Linker procedure
@@ -186,7 +192,7 @@ all: $(BIOSSRC) Debug Release
 
 $(BIOSSRC): $(BIOSTCF)
 	@echo Building DSP/BIOS file...
-	@$(BASE_SABIOS)/xdctools/$(TCONF) $(BFLAGS) $< 0
+	@$(TCONF) $(BFLAGS) $< 0
 
 #   ----------------------------------------------------------------------------
 #   Building Debug... 
